README.md

```diff
@@ -37,6 +41,9 @@ The following attributes are available for the `viam-labs:mlmodel:onnx-cpu` ML m
 | ------- | ------ | ------------ | ----------- |
 | `model_path` | string | **Required** | The full path to the ONNX file. |
 | `label_path` | string | Optional | The full path to the names of the category labels. |
+| `num_threads` | int | Optional | The number of threads. Must be 1. |
+| `package_reference` | string | Optional | The package of the model to deply. For example: "viam-soleng/face-detector-onnx". |
```

Comment:
Do you know what the `package_reference` is? We don't use the field in the config of the module, the module would only know about the `model_path`, which should hold the path to where the model was deployed if the user used the registry to get their model

Reply:
right that's a problem in the UI as well then. I'll tag you in the ticket